Windows Community Toolkit.sln

```diff
@@ -658,6 +660,30 @@ Global
 		{D4D78CBA-B238-4794-89A0-4F1A2D8FEA97}.Release|x64.Build.0 = Release|Any CPU
 		{D4D78CBA-B238-4794-89A0-4F1A2D8FEA97}.Release|x86.ActiveCfg = Release|Any CPU
 		{D4D78CBA-B238-4794-89A0-4F1A2D8FEA97}.Release|x86.Build.0 = Release|Any CPU
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Debug|Any CPU.ActiveCfg = Debug|Win32
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Debug|Any CPU.Build.0 = Debug|Win32
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Debug|ARM.ActiveCfg = Debug|ARM
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Debug|ARM.Build.0 = Debug|ARM
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Debug|x64.ActiveCfg = Debug|x64
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Debug|x64.Build.0 = Debug|x64
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Debug|x86.ActiveCfg = Debug|Win32
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Debug|x86.Build.0 = Debug|Win32
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Native|Any CPU.ActiveCfg = Release|x64
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Native|Any CPU.Build.0 = Release|x64
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Native|ARM.ActiveCfg = Release|ARM
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Native|ARM.Build.0 = Release|ARM
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Native|x64.ActiveCfg = Release|x64
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Native|x64.Build.0 = Release|x64
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Native|x86.ActiveCfg = Release|Win32
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Native|x86.Build.0 = Release|Win32
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Release|Any CPU.ActiveCfg = Release|Win32
+		{D4C7AC54-826A-4412-8461-A7C7FD86534B}.Release|Any CPU.Build.0 = Release|Win32
```

Comment:
Try removing this line - should solve the build issue

Reply:
Tried that and it didn't work, I'm still getting:


       "C:\Users\junma\source\repos\WindowsCommunityToolkit\Windows Community Toolkit.sln" (Build target) (1) ->
       "C:\Users\junma\source\repos\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.WinRT\Microsoft.Toolkit.Uwp.UI.Controls.WinRT.vcxproj" (default targ
       et) (2) ->
       (ResolveNuGetPackageAssets target) ->
         C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\Microsoft\NuGet\15.0\Microsoft.NuGet.targets(186,5): error : Your project does not
       reference "UAP,Version=v10.0" framework. Add a reference to "UAP,Version=v10.0" in the "TargetFrameworks" property of your project file and then re-run NuG
       et restore. [C:\Users\junma\source\repos\WindowsCommunityToolkit\Microsoft.Toolkit.Uwp.UI.Controls.WinRT\Microsoft.Toolkit.Uwp.UI.Controls.WinRT.vcxproj]

Reply:
I'm trying to deal with an issue now where if a C# project takes the nuget package, things are fine. The Generic.xaml that I have in my project gets pushed to the C# project and the control works as expected.

In C++/CX and winrt projects, the Generic.xaml isn't placed with the application so my control never applies anything on "ApplyTemplate" so things basically don't work. If I copy/paste my default style into the C++ project things are fine. There appears to be a difference between how C# and C++ UWP projects work.

Has anyone dealt with this before? I'm having a hard time trying to figure out why C++/C# UWP projects treat referenced XAML controls differently.